buyer-api/test/operations/notarizationResultReception.test.js

```diff
@@ -0,0 +1,35 @@
+import test from 'ava';
+import { addTransactionJob } from './notarizationResultReception.mock';
+import { notarizationResultReception } from '../../src/operations/notarizationResultReception';
+import {
+  someNotarizationRequest,
+  someNotarizationResult,
+  someNotarizationResultWithDuplicatedAddresses,
+  someNotarizationResultWithNonRequestedAddresses,
+} from './notarizationResultReception.fixture';
+
+const it = test.serial;
+
+it('call notarizationResultReception', async (assert) => {
```

Comment:
Test description should be about what this test is trying to assert.
(And by convention are written on simple present)